lib/Factor.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+// This module deals with getting factors
+
+const Factor = {};
+
+Factor.getPrimeFactors = function(number){
+  let factors = [];
+  if (number < 0) {
+    factors = [-1];
+    factors = factors.concat(Factor.getPrimeFactors(-1 * number));
+    return factors;
+  }
+
+  const root = Math.sqrt(number);
+  let candidate = 2;
+  if (number % 2) {
+    candidate = 3; // assign first odd
+    while (number % candidate && candidate <= root) {
+      candidate = candidate + 2;
+    }
+  }
+
+  // if no factor found then number is prime
+  if (candidate > root) {
+    factors.push(number);
+  }
+  else {
+    // if number isn't prime factor make recursive call
```

Comment:
ohhh, got it

can you make the comment "if the factor isn't prime, make a recursive call to add its prime factors" or something like that? I thought number was `number`